app/models/course/condition.rb

```diff
@@ -5,4 +5,6 @@ class Course::Condition < ActiveRecord::Base
   belongs_to :conditional, polymorphic: true
 
   delegate :satisfied_by?, to: :actable
+
+  NAMES = ['achievement', 'assessment', 'level']
```

Comment:
I need this to iterate through all conditions to select in `New...` dropdown button [here](https://github.com/Coursemology/coursemology2/pull/685/files#diff-c7c8bc68e2f0743a235559f2b9a11fbeR11)